code/game/objects/machinery/smartgun_mount.dm

```diff
@@ -398,6 +399,7 @@
 		A = target
 	proj_to_fire.setDir(dir)
 	proj_to_fire.def_zone = pick("chest","chest","chest","head")
+	proj_to_fire.projectile_iff = iff_signal
```

Comment:
For some reason, the mounted HMG does not fire any smartgun bullets.

Reply:
If you are referring to the official server, I have updated the PR and it should be fixed once the testmerged PRs are updated and this one has tmc tag.

Let me know in case you are instead testing the latest version from github as on my tests the TL-102 should have IFF.